### ISSUE TITLE
What is a Null route

### ISSUE THREAD

**Issue by adrianhunter** [arunoda/meteor-apm-client#50]
When i go to Pub/Sub Overview and look at the routes Breakdown, i see a big amount of "Null" routes. What is a Null route?

**Comment by arunoda** [comment]
If you are doing subscriptions out from Iron Router or making them with Fast Render.

**Comment by dandv** [comment]
Same question here. Would be nice to explain null routes better in the UI. Maybe a tooltip, but perhaps a better explanation is needed: in my case (app `6kpgrQKaEf52bsb8z`), all subscriptions are managed through Iron Router and I don't use Fast Render yet. I still see some real routes, as well as some "Null" routes:

![image](https://cloud.githubusercontent.com/assets/33569/3130414/b69c73e2-e7f7-11e3-95d1-3aa312bd6608.png)

What exactly are the "Null" routes? All iron-router routes must have names.

**Comment by arunoda** [comment]
Good point. There are some other publications meteor sends to the client
with out an route. Everytime when you create an session. That's why they
are displayed allocated in the Null.

---

Arunoda Susiripala

@arunoda http://twitter.com/arunoda
 http://gplus.to/arunodahttps://github.com/arunoda
http://www.linkedin.com/in/arunoda
